src/FeeCollector.sol

```diff
@@ -13,19 +13,15 @@ struct FeeRecipient {
     uint16 percentageBps;
 }
 
-struct PositionData {
-    Party party;
-    FeeRecipient[] recipients;
-}
-
 contract FeeCollector is IERC721Receiver {
     INonfungiblePositionManager public immutable POSITION_MANAGER;
     address payable public immutable PARTY_DAO;
     IWETH public immutable WETH;
 
     uint16 public partyDaoFeeBps;
 
-    mapping(uint256 tokenId => PositionData) public getPositionData;
+    mapping(uint256 tokenId => FeeRecipient[] recipients)
+        private _feeRecipients;
```

Comment:
Made this private because Solidity behaves weirdly with auto-generated getter function for mappings to arrays of structs